### DISCUSSION TITLE
ImportError: could not import ellc

### DISCUSSION THREAD

**Discussion by famc2** [phoebe-project/phoebe2#584] [Questions]
Hello,

>>> phoebe.__version__
'2.3.59'

I follow this example:
http://phoebe-project.org/docs/2.3/examples/inverse_paper_examples

after this lines:
b.add_compute('ellc', compute='fastcompute')
b.set_value_all('ld_mode', 'lookup')
b.run_compute(compute='fastcompute')

I got error:
ImportError: could not import ellc

no result after googling

thank you!

**Comment by kecnry** [comment]
All of the [alternate backends](http://phoebe-project.org/docs/latest/tutorials/alternate_backends) are soft-dependencies - meaning that the other packages are not installed with phoebe automatically, but are required in order to use the backend.  In this case, you're trying to run the ellc backend, so need to install [ellc](https://github.com/pmaxted/ellc) (which is available from pip).

**Reply by kecnry** [reply]
I created a [docs issue](https://github.com/phoebe-project/phoebe2-docs/issues/26) to update the notebook to make this more clear.  Thanks for pointing this out!

**Reply by famc2** [reply]
OK, thank you
